build/.webignore

```diff
@@ -25,3 +25,5 @@ xterm-addon-unicode11/out/**
 
 xterm-addon-webgl/src/**
 xterm-addon-webgl/out/**
+
+tas-client/**
```

Comment:
This change can be removed now.

Reply:
yep removed, the rules at the top of this file cover all the files in the tas-client-umd module